packages/components/src/Chip/Chip.tsx

```diff
@@ -41,19 +42,21 @@ export const Chip = ({
     label,
     heading,
   );
+  const Tag = onClick ? "button" : "div";
 
   return (
-    <Tooltip message={tooltipMessage}>
-      <button
+    <Tooltip message={tooltipMessage} setTabIndex={false}>
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
           onClick && onClick(value, ev)
         }
         tabIndex={tabIndex}
         onKeyDown={onKeyDown}
-        aria-label={ariaLabel || label}
+        aria-label={ariaLabel}
```

Comment:
Just a question, was a particular reason not to default this to the label if an `ariaLabel` wasn't present?

No change needed if it was intentional